src/account-mgr.cpp

```diff
@@ -119,6 +120,8 @@ int AccountManager::saveAccount(const Account& account)
 
 int AccountManager::removeAccount(const Account& account)
 {
+    RepoService::instance()->onAccountRemoval(account);
+
```

Comment:
What about adding a new signal "accountRemoved" and let `ReposService` listen to this signal?